js/modules/k6/http/request_test.go

```diff
@@ -1641,6 +1641,8 @@ func checkErrorCode(t testing.TB, tags *stats.SampleTags, code int, msg string)
 }
 
 func TestErrorCodes(t *testing.T) {
+	// lookup dafsgdhfjg on 127.0.0.53:53: server misbehaving
```

Comment:
can you try to change the domain to `s.com` or something else that will not be existent but looks more like a domain as this seems like github action DNS problem

Reply:
Thanks, that worked! 🎉 

I enabled the test in c9b1b9ba.